taker/src/main.rs

```diff
@@ -383,9 +384,9 @@ async fn main() -> Result<()> {
                 routes::post_withdraw_request,
             ],
         )
-        .register("/api", rocket::catchers![rocket_basicauth::unauthorized])
+        .register("/api", default_catchers())
```

Comment:
Does this indirection actually add value?

Reply:
yup, this shows that we use the same catchers on both sides, otherwise you need manual inspection to prove it. It is less risk of maintenance burden, less likely to forget to add a catcher on the other side when it gets added in one place.